project/AppDependencies.scala

```diff
@@ -3,36 +3,39 @@ import sbt.*
 
 object AppDependencies {
 
-  private val catsVersion = "2.8.0"
-  private val bootstrapVersion = "7.22.0"
+  private val catsVersion = "2.9.0"
+  private val bootstrapVersion = "7.23.0"
+  private val xalanVersion = "2.7.3"
 
   val compile: Seq[ModuleID] = Seq(
     "uk.gov.hmrc"             %% "bootstrap-backend-play-28"  % bootstrapVersion,
     "com.dmanchester"         %% "playfop"                    % "1.0",
     "net.sf.barcode4j"        %  "barcode4j"                  % "2.1",
     "net.sf.barcode4j"        %  "barcode4j-fop-ext"          % "2.1",
-    "com.lucidchart"       %% "xtract"                         % "2.2.1"
+    "com.lucidchart"          %% "xtract"                     % "2.2.1",
+    "xalan"                   %  "xalan"                      % xalanVersion,
```

Comment:
is this supposed to be removed?

Reply:
"Xalan has been retired and no security fixes are being applied anymore, dependency should be removed"

Reply:
I'll try again tomorrow to see if this can be done without Xalan. Without it I get `NoClassDefFoundError` when running the application